meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -84,11 +88,16 @@ sync -f "$SYSROOT"
 mv "$SYSROOT/counter.new" "$SYSROOT/counter"
 sync -f "$SYSROOT"
 
-# Run any defined hooks
-for hook in "/etc/hostapp-update-hooks.d/"*; do
-	[ -e "$hook" ] || break
-	"$hook" "$CONTAINER_ID"
-done
+# Run all the defined hooks in the host OS we update to
+if [ "$hooks" = 1 ]; then
+	hooks_list_new="/tmp/$(basename $hooks_list).new"
```

Comment:
Why do we need to have a list of hooks in a text file instead of reading the directory? Also, would it make sense to use `balena run --rm $HOSTAPP_IMAGE run_hooks` with some bind mounts set up and let the new version do its thing?

Reply:
Yes. I went down the `balena run` path.

Reply:
So why the list of hooks in a text file?

Reply:
The idea of a list of hooks in a text file was abandoned.